### PR TITLE
DIG for tokensdk

### PR DESCRIPTION
**Pull request by alexandrosfilios**

### CODE REVIEW

File: token/dig/sdk.go [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+	fabricsdk "github.com/hyperledger-labs/fabric-smart-client/platform/fabric/dig"
+	view2 "github.com/hyperledger-labs/fabric-smart-client/platform/view"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/driver"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/services/flogging"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/services/kvs"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/services/tracing"
+	"github.com/hyperledger-labs/fabric-token-sdk/token"
+	core2 "github.com/hyperledger-labs/fabric-token-sdk/token/core"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/core/common/logging"
+	_ "github.com/hyperledger-labs/fabric-token-sdk/token/core/fabtoken/driver"
```
Comment (adecaro) [inline token/dig/sdk.go:25]: the other sdk.go doesn't have any blind imports. Why do we need them here?

File: token/dig/sdk.go [hunk 2]
```diff
@@ -0,0 +1,229 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package sdk
```
Comment (adecaro) [inline token/dig/sdk.go:7]: can we move this package under sdk?

**Review by adecaro — CHANGES_REQUESTED** [review]

File: token/dig/sdk.go (hunk 1 above)
Comment (alexandrosfilios) [reply token/dig/sdk.go]: Oh no we don't need them. This version was based on a previous version of the sdk that had the blind imports. Will remove them.

**Review by adecaro — APPROVED** [review]
LGTM

**Pull request merged**